libs/ledger-live-common/src/env.test.ts

```diff
@@ -0,0 +1,8 @@
+import { getEnv } from "@ledgerhq/live-env";
+
+// @ts-expect-error sanity check on getEnv() not accepting incorrect key names
+getEnv("yolo");
+
+test("typecheck env", () => {
+  expect(true).toBe(true);
+});
```

Comment:
I don't get what these are testing?

Reply:
we are using `// @ts-expect-error` on the first line, which basically EXPECT typescript to throw an error. If, by some changes, we break the type inference on `getEnv("")` and `yolo` now works, the comment itself will throw an error during typechking

Reply:
I think you can instead move this file to `src/typescript-tests/env.ts`

the goal is to do a typescript test, this one:

```
// @ts-expect-error
getEnv("yolo");
```

to actually non regress on what we just fixed here ;)